### PR TITLE
Feature/JA-28 - Add data structures of: Ad, Schedule and User

### PR DESCRIPTION
**Pull request by skrawus**

### CODE REVIEW

**Review by GrzegorzKmiecinski — APPROVED** [review]

File: TutorLizard.BusinessLogic/Models/Schedule.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TutorLizard.BusinessLogic.Models;
+
+public class Schedule
+{
+    public int ScheduleId { get; set; }
```
Comment (Zjyslav) [inline TutorLizard.BusinessLogic/Models/Schedule.cs]: Please change ScheduleId to Id

**Review by Zjyslav — CHANGES_REQUESTED** [review]
Please change Schedule Id

File: TutorLizard.BusinessLogic/Models/Schedule.cs (hunk 1 above)
Comment (skrawus) [reply TutorLizard.BusinessLogic/Models/Schedule.cs]: changed

Comment (Zjyslav) [reply TutorLizard.BusinessLogic/Models/Schedule.cs]: Thank you.

**Review by Zjyslav — APPROVED** [review]

**Pull request merged**